### ISSUE TITLE
[BUG] Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by Xandromedia** [Chia-Network/chia-blockchain-gui#1876]
### What happened?

Need help! GUI crashed by farming & doesn´t see on pool! 

### Version

1.8.2

### What platform are you using?

Linux

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
Cannot read properties of undefined (reading 'length')

URL
#/dashboard/vc

Stacktrace
at webpack://@chia-network/gui/src/components/vcs/VCList.tsx:303:23 renderBadgeContainer
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:157:136 Ch
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:180:153 oi
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:269:342 a=b.pendingProps;f=e._init;e=f(e._payload);b.type=e;f=b.tag=hk(e);a=lg(e,a);switch
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:346 Pj
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:277 Pj
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:137 Pj
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:243:162 Lj
at webpack://@chia-network/gui/node_modules/react-dom/cjs/react-dom.production.min.js:123:114 jg
at webpack://@chia-network/gui/node_modules/scheduler/cjs/scheduler.production.min.js:18:342 unstable_next
```

**Comment by seeden** [comment]
Fixed via https://github.com/Chia-Network/chia-blockchain-gui/pull/1888

**Comment by paninaro** [comment]
Fix is in 2.0.0-rc1 and later
